packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -81,10 +80,28 @@ const startTrRoutingProcess = async (
     return processStatus;
 };
 
+const getCacheAllScenarios = () => {
```

Comment:
Could we make a class that manage access to the config (a TrRoutingConfig or something like that.) Eventually, it could be merge to a more global config thing, but to group all config access in one places. 
Could all be static function. 

Optionaly, for the lazy path, Can we name the function something with Config in the name.